cypress/fixtures/singleBeaconRegistration.ts

```diff
@@ -0,0 +1,90 @@
+import { Registration } from "../../src/entities/Registration";
+
+export const singleBeaconRegistration: Registration = {
```

Comment:
Chose to make this fixture a `.ts` file rather than a `.json` so the type system keeps the fixture correct.